exporters/zipkin/CMakeLists.txt

```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 
 include_directories(include)
-find_package(CURL REQUIRED)
```

Comment:
Not related to this PR, but while you are on it, can you modify the copyright header here to be consistent with other places.

Reply:
Done.